subprojects/packagefiles/spdlog/meson.build

```diff
@@ -16,6 +16,7 @@ spdlog_compile_args = []
 
 if meson.get_compiler('cpp').has_header_symbol('format', '__cpp_lib_format', required: get_option('std_format'))
   spdlog_compile_args += '-DSPDLOG_USE_STD_FORMAT'
+  fmt_dep = depemndency('', required: false)
```

Comment:
Sorry to interrupt, but isn't `depemndency` spelled wrong here?
Shouldn't it be `dependency`

Reply:
@Totto16 You're absolutely right. Not sure how this typo went it before I submit it.